package.json

```diff
@@ -161,4 +160,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
```

Comment:
Won't let me leave a comment on a line that wasn't changed but:

While we're here, what are your thoughts on removing the `build` script from the package.json `scripts`.

`build:library` is what's used in all the CI/CD, so we're safe to remove `build`. Might help avoid confusion in the future. If it was useful to your dev work on this repo, we should definitely keep it but, it seems like `start` is really the dev mode. Wondering if you used `start` over `build`, and in that case, if can remove `build`.

^ Non blocking comment

Reply:
Yeah, agreed. I don't see why it would be needed, and if it is - someone could easily add a build back in by googling webpack build steps